### ISSUE TITLE
GCC path without bin

### ISSUE THREAD

**Issue by josesimoes** [carlosperate/arm-none-eabi-gcc-action#21]
Hey!

Thank you for this handy git action. 👏🏻 
Something I've noticed when trying to use it: the export path for the toolchain includes the 'bin' part at the end. This makes it different from the usual GCC path that one finds pretty much everywhere else, including the path that GCC adds to the PATH environment variable.
I get it that this could be a breaking change for some folks at this time, but I would suggest that you consider this.

**Comment by carlosperate** [comment]
Thanks for the suggestion @josesimoes!

In my experience the Arm gcc installer only adds the bin directory to the PATH, I just had a look in my macOS and Windows installations and that's all I've got.
I have used some Makefiles that then required some config to point to the top level directory, so I can see that being useful as well, but things like CMake are usually fine with just the default bin directory being in the PATH.

To better understand the usage, do you have any good examples of build systems that expect the top level directory instead of the bin?

**Comment by loopj** [comment]
Ran into this myself today, here's an example of a "build system" that expects the top level directory:
https://docs.silabs.com/simplicity-studio-5-users-guide/latest/ss-5-users-guide-tools-slc-cli/03-usage#slc-cli-configuration

I'd personally propose we add an additional env variable for the path without bin, that way it isnt a breaking change, but it solves usecases where the base path is needed.

**Comment by carlosperate** [comment]
Yes, I think that's a fair request. Would you like to raise a PR @loopj ?
